Gemfile.lock

```diff
@@ -229,7 +254,7 @@ GEM
     netrc (0.11.0)
     no_proxy_fix (0.1.2)
     notify (0.5.2)
-    octokit (4.18.0)
+    octokit (4.22.0)
```

Comment:
when I did this earlier it broke Danger after merging. I think there's something that changed with the way auth works between these two versions https://github.com/CocoaPods/CocoaPods/pull/11225

Reply:
ah let me fix that.

Reply:
updated.